README.md

```diff
@@ -59,6 +59,21 @@ Generate coverage reports for the whole project:
 
     ./gradlew reportCoverage
 
+### Generating an API compatibility report between trunk and your local branch ###
```

Comment:
This block seems misplaced. It looks like you wanted to put this after description of coverage tasks, but `./gradlew clients:reportCoverage` is after this one.

Reply:
Thanks Andras, I've addressed this and moved the section after the coverage section.